app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -366,9 +367,15 @@ fun LazyListState.isScrolledToEnd(): Boolean {
     return out
 }
 
-fun openLink(url: String, ctx: Context) {
-    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
-    ctx.startActivity(intent)
+fun openLink(url: String, ctx: Context, useCustomTab: Boolean) {
+    if (useCustomTab) {
+        val intent = CustomTabsIntent.Builder()
+            .build()
+        intent.launchUrl(ctx, Uri.parse(url))
```

Comment:
These both did the exact same thing in my testing, just loaded up my external browser.

Seems more complicated than this. Check here:

https://stackoverflow.com/a/70514350/1655478

Reply:
I got some weird results in my testing, but it does appear to work in both the emulator and on my Pixel 5, using chrome and firefox as providers, respectively. The first couple of times I tried using it, it would open in the default browser, which is a documented fallback mode if no installed browser provides custom tabs. But it is behaving correctly now. I wish I had a better understanding of this behavior.

Reply:
Mmmk. Maybe its just a result of my default browser not supporting custom tabs.